### ISSUE TITLE
[Medium]: Color Contrast: Insufficient color contrast for control.

### ISSUE THREAD

**Issue by SumitDiyora** [Expensify/App#9544]
## Action Performed:
1. Using Chrome, open URL: staging.new.expensify.com
2. Select any chat.
3. Click on any earlier attachment or Click on the grey colored plus icon and select "Add attachment",Upload any picture and click on send to upload attachment. 
5. Click on the 'Download' icon.
6. Observe that the analyser reports that the contrast ratio is not sufficient.

## Expected Result:
The "Download icon" control (which is available in tool-tip after selecting any attachment) should have a contrast ratio against the control background of 3.0:1 or greater.

## Actual Result:
Background/foreground color combinations with an insufficient contrast ratio was found on control. For instance, the #03D47C "Download icon" control (which is available in tool-tip after selecting any attachment) has an insufficient color contrast ratio of 2.0.1 with a #FFFFFF background.

## Workaround:
Yes

## Area issue was found in:
Chat section

## Failed WCAG checkpoints
1.4.11

## User impact:
It will be a bit confusing or there will be difficult to perceive the content for the low vision users if the color contrast ratio does not meet the minimum requirement.

## Suggested resolution:
Ensure that each text/background color combination presents a contrast ratio of at least 3.0:1.

## Platform:
- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:*1.1.78-0*
**Reproducible in staging?:*Yes*
**Reproducible in production?:*Yes*
**Issue reported by:*Sumit*

![color contrast of download icon](https://user-images.githubusercontent.com/98817464/175268888-9d9ed4bc-9a40-4c4b-868a-e761f9282e84.jpg)
